src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsTable.tsx

```diff
@@ -164,11 +213,27 @@ const ChangeRequestsTable: React.FC = () => {
             reviewer: fullNamePipe(v.reviewer)
           })) || []
         }
-        columns={columns}
+        columns={windowSize < 900 ? smallColumns : columns}
         getRowId={(row) => row.crId}
-        sx={{ background: theme.palette.background.paper }}
+        sx={{
```

Comment:
Are we able to make all this into its own design class? Might wanna use the same design/looks on other things

Reply:
that's a good point, but the datagrid is pretty involved, so I think it would be good to do that if/when we end up wanting to use it somewhere else